tests/main/views/test_suppliers.py

```diff
@@ -13,68 +12,70 @@
 
 
 class TestGetSupplier(BaseApplicationTest, FixtureMixin):
+    supplier = supplier_id = None
+
     def setup(self):
         super(TestGetSupplier, self).setup()
 
         with self.app.app_context():
-            payload = load_example_listing("Supplier")
+            payload = load_example_listing("supplier_creation")
```

Comment:
Yeah, we used the `Supplier` fixture file in loads of places because we could "PUT" it into the database and then hardcode the id everywhere thereafter.  

I agree that it's probably possible to turn `"supplier_creation"` into `"Supplier"` (and drop the existing supplier), but at a certain point, you just stop going down these rabbit holes.

(I'll have a look though, see how hard it is.)